packages/e/era-widget.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "era-widget",
+  "description": "Build custom widget for e-ra.io platform",
+  "filename": "index.js",
+  "keywords": [
+    "e-ra",
+    "era",
+    "widget"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/eoh-public/era-widget",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eoh-public/era-widget.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@eohjsc/era-widget",
+    "fileMap": [
+      {
+        "basePath": "dist",
```

Comment:
Looking at `https://www.npmjs.com/package/@eohjsc/era-widget?activeTab=code`, your package does not contain a `dist` directory.